controllers/toiletController.js

```diff
@@ -73,18 +73,36 @@ function create(req, res) {
 
 
 function update(req,res) {
-  db.Toilet.findByIdAndUpdate(req.body.id, {new:true}, (err, toilet) => {
-    console.log(req.body);
-    toilet.name = req.body.name;
-    toilet.address = req.body.address;
-    toilet.price = req.body.price;
-    toilet.public = req.body.public;
-        // toilet.rating = req.body.rating,
-    toilet.amount = req.body.amount;
-    toilet.save();
-    res.sendStatus(200);
+    console.log(req.body.availability);
+    let toiletRatingSum = 0;
+    let toiletLength = 0;
+    let averageRating;
+    db.Review.find({toilet: req.body.id}, (err, toiletRating) => {
+        toiletRating.forEach(function(sumToiletRating) {
+            toiletLength += 1;
+            toiletRatingSum += sumToiletRating.rating
+    })
+    if (toiletLength === 0) {
+            averageRating = 3;
```

Comment:
Is there a reason this defaults to 3? Shouldn't it default to the toilet's initial rating?